modules/virt-regional-dr-odf.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * virt/backup_restore/virt-disaster-recovery.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="regional-dr-odf_{context}"]
+= Regional-DR for {rh-storage-first}
+
+{VirtProductName} supports the link:https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/4.17/html-single/configuring_openshift_data_foundation_disaster_recovery_for_openshift_workloads/index#rdr-solution[Regional-DR solution for {rh-storage}], which provides asynchronous data replication at regular intervals between managed {VirtProductName} clusters installed on primary and secondary sites.
+
+.Regional-DR differences
+* Regional-DR supports higher network latency between the primary and secondary sites.
+* Because Regional-DR uses RBD snapshots to replicate data, applications must be resilient to small variances between VM disks. You can avoid this scenario by using single disk VMs.
```

Comment:
```suggestion
* Because Regional-DR currently uses RBD snapshots to replicate data, applications must be resilient to small variances between VM disks. You can avoid this scenario by using single disk VMs.
```

@mulbc would this be ok?

Reply:
I don't think this requirement will change. It is inherent with regional-DR type setups. The fail over site will always be behind the primary site for a period of time.

Reply:
@ousleyp - works for me
@awels - yes, it's async replication, but what will change is that we will eventually add consistency groups, which will allow us to remove sync gaps between 2 or more volumes in the sync set

Reply:
@mulbc I had some second thoughts about the initial phrasing I suggested. Updating the doc to the following (please let me know if you have concerns about hte phrasing):

> Regional-DR uses RBD snapshots to replicate data asynchronously. Currently, your applications must be resilient to small variances between VM disks. You can prevent these variances by using single disk VMs.